aws/structure.go

```diff
@@ -1906,6 +1906,21 @@ func sortInterfaceSlice(in []interface{}) []interface{} {
 	return b
 }
 
+func sortIntInterfaceSlice(in []interface{}) []int {
```

Comment:
Functions like these are generally discouraged as we can ignore ordering if the API ignores ordering. I think when the schema adjustments are made (e.g. `failover_criteria` is TypeList + MaxItems: 1 and `status_codes` is TypeSet), this will be unnecessary. 👍